src/statespace/dart/SE3Joint.cpp

```diff
@@ -6,8 +6,8 @@ namespace dart {
 
 //=============================================================================
 SE3Joint::SE3Joint(::dart::dynamics::FreeJoint* _joint)
-  : JointStateSpace(_joint)
-  , SE3()
+  : SE3()
+  , JointStateSpace(_joint) // Order of class members to be maintained
```

Comment:
This comment is misleading. `SE3` and `JointSubSpace` are super classes, not class members.